packages/client/src/generation/ts-builders/AnyDeclarationBuilder.ts

```diff
@@ -1,6 +1,7 @@
+import { ClassDeclaration } from './Class'
 import { ConstDeclaration } from './ConstDeclaration'
 import { InterfaceDeclaration } from './Interface'
 import { TypeDeclaration } from './TypeDeclaration'
 
-// TODO: enum, class
-export type AnyDeclarationBuilder = TypeDeclaration | ConstDeclaration | InterfaceDeclaration
+// TODO: enum
+export type AnyDeclarationBuilder = TypeDeclaration | ConstDeclaration | InterfaceDeclaration | ClassDeclaration
```

Comment:
I think `Export` and `Import` would also belong here.

Reply:
If we allow `Export` in the list, then it means one could end up with `export export...`?